mobile/native/src/event/mod.rs

```diff
@@ -27,6 +27,7 @@ pub fn publish(event: &EventInternal) {
 pub enum EventInternal {
     Init(String),
     Log(String),
+    NodePubkey(String),
```

Comment:
🔧

I don't think we need to define a whole `Event` for this - calling `api::get_node_id().0 in the test` should suffice! (this is how we can acess the `SyncReturn` in rust tests. 
This would also be the same way that we are retrieving the node id in the app, so we will slightly increase test coverage 😉

this should decrease the size of this PR substantially.